packages/runtime/container-runtime/src/summary/summarizerNode/summarizerNode.ts

```diff
@@ -165,26 +180,24 @@ export class SummarizerNode implements IRootSummarizerNode {
 		trackState: boolean = true,
 		telemetryContext?: ITelemetryContext,
 	): Promise<ISummarizeResult> {
+		// Set to wipSummarizeCalled true to represent that current node was included in the summary process.
+		this.wipSummarizeCalled = true;
+
 		// If trackState is false, call summarize internal directly and don't track any state.
 		if (!trackState) {
 			return this.summarizeInternalFn(fullTree, trackState, telemetryContext);
 		}
 
 		// Try to reuse the tree if unchanged
 		if (this.canReuseHandle && !fullTree && !this.hasChanged()) {
-			const latestSummary = this._latestSummary;
-			if (latestSummary !== undefined) {
-				this.wipLocalPaths = {
-					localPath: latestSummary.localPath,
-					additionalPath: latestSummary.additionalPath,
-				};
+			if (this._lastSummaryReferenceSequenceNumber !== undefined) {
 				this.wipSkipRecursion = true;
 				const stats = mergeStats();
 				stats.handleNodeCount++;
 				return {
 					summary: {
 						type: SummaryType.Handle,
-						handle: latestSummary.fullPath.path,
+						handle: this._summaryHandleId.toString(),
```

Comment:
```suggestion
						handle: this.summaryHandleId,
```

Reply:
@agarwal-navin --- the type of the handle is a string in `ISummaryHandle`.

Reply:
The type of `summaryHandle` in this class is also string. it does `this._summaryHandleId.toString()` for you so there is no need to do it here again.